specifications/credential.issuance.protocol.md

```diff
@@ -94,13 +94,16 @@ Self-Issued ID Token to provide the pre-authorization code to the issuer.
+|              |                                                                                                                                                                                                                                                                                                                                                                |
+|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Schema**   | [JSON Schema](./resources/issuance/credential-request-message-schema.json)                                                                                                                                                                                                                                                                                     |
+| **Required** | - `@context`: Specifies a valid Json-Ld context ([[json-ld11]], sect. 3.1).                                                                                                                                                                                                                                                                                    |
+|              | - `type`: A string specifying the `CredentialRequestMessage` type                                                                                                                                                                                                                                                                                              |
+|              | - `holderPid`: A string corresponding to the request id on the Holder side type                                                                                                                                                                                                                                                                                |
+|              | - `credentials`: a JSON array of objects, each containing a `format`, which is A JSON string <br/>that describes the format of the credential to be issued <br/>, a `type`: A JSON array of strings that specifies the VC type being requested and <br/>optionally a `credentialSubject` property that is a populated object according to the `credentialType` |
+
+When receiving a `credentials.credentialSubject` object with a `credentialSubject` property that the [=Issuer=] refuses
+to sign, the server MUST reject the entire message altogether.
```

Comment:
Should we define a specific HTTP response?

Reply:
I think the HTTP response will be 2xx in any case - the issuance request would be rejected via the [request status API](https://eclipse-dataspace-dcp.github.io/decentralized-claims-protocol/v1.0-RC2/#credential-request-status-api), wouldn't it?

Clarifying the phasing.

Reply:
I think we should always have the option to reject it immediately. Otherwise, it opens the issuer up to a potential DDOS attack because the request will need to be tracked.

Reply:
True - in that case I'd argue that the clarified phrasing is sufficient [1]. It leaves open to immediately return HTTP4xx or fail the process via the credential request API. Do you see a case where this behavior (fail early or fail late) is relevant for interop?

`When receiving a `credentials.credentialSubject` object with a `credentialSubject` property that the [=Issuer=] refuses
to sign, the server MUST reject the issuance request altogether. `

Reply:
I think we should define these error cases:

1. The issuer does not support the capability at all - fail fast with a specific return code
2. The issuer does not support the capability for a particular credential type - fail fast with a specific return code
3. The issuer does not support adding a specific property for the type  - fail fast with a specific return code
4. Invalid data is supplied - either fail fast with a specific return code or later through the status API